types/jsforce/connection.d.ts

```diff
@@ -65,6 +65,12 @@ export interface UserInfo {
     url: string;
 }
 
+// IdentityInfo is a large object of identity information that is not typed in jsforce documentation. 
+// https://jsforce.github.io/jsforce/doc/Connection.html#identity
+export interface IdentityInfo {
+    [key:string]: any
+}
+
```

Comment:
I opted to leave this loose for now due to the size of the IdentityInfo object and lack of documentation in jsforce docs.

Reply:
For reference, here’s article about the identity endpoint https://help.salesforce.com/articleView?id=remoteaccess_using_openid.htm&type=5&sfdcIFrameOrigin=null

Reply:
Thanks @douglascayers. I guess my only question is if typing this out is necessary for this PR to go in? Or something that can be done in a follow up PR.

Reply:
@sacummings91 I think it's worth while to finish up the typings since we already have a PR in flight. I've prepared [this branch](https://github.com/douglascayers/DefinitelyTyped/tree/pr-43448-jsforce-types) that includes your changes plus the identity properties.

If you don't mind, please merge my branch into yours then push back up. Thanks!

_To merge my changes into your branch:_
```
git remote add douglascayers https://github.com/douglascayers/DefinitelyTyped.git
git fetch douglascayers
git merge douglascayers pr-43448-jsforce-types
git push
```

Reply:
Hey @douglascayers. I went ahead and merged your changes into my branch. Also, thank you for going ahead and doing that. I appreciate it.

Reply:
Thanks @sacummings91

Reply:
No problem @douglascayers. What steps from here do we need to get this PR in?

Reply:
We’re waiting on one of the other approves to take action or for the Typescript team to engage. I think we are still in the waiting period per https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43448#issuecomment-605507026